scripts/build.js

```diff
@@ -1,42 +1,38 @@
-const fs = require('fs')
-const rp = require('request-promise-native')
-const xlsx = require('xlsx')
+const fs = require('fs');
+const rp = require('request-promise-native');
+const xlsx = require('xlsx');
 
-const url = 'http://www.iso15022.org/MIC/ISO10383_MIC.xls'
-const encoding = null
+const url = 'http://www.iso15022.org/MIC/ISO10383_MIC.xls';
+const encoding = null;
 
 function exit(err) {
   if (err) {
     /* eslint-disable no-console */
-    console.log(err)
+    console.log(err);
     /* eslint-enable no-console */
-    process.exit(1)
+    process.exit(1);
   } else {
-    process.exit(0)
+    process.exit(0);
   }
 }
 
 rp({ url, encoding })
   .then(res => {
-    try {
```

Comment:
The reason I put the try/catch block here is if `JSON.stringify` or `fs. writeFileSync` throws, the `exit` function would handle it. Of course it doesn't really matter because the script will exit anyways and the error output will be logged to the console, but I thought it was more "procedural" that way.